gyucheol/2일차/두-정수-사이의-합.js

```diff
@@ -0,0 +1,31 @@
+function solution(a, b) {
+    let sum = 0;
+    if(a >= b){
+        for(let i = b; i <= a; i++){
+            sum += i; // i가 1씩 계속 증가하니까 0 + n + m + ...
+        }
+    }else{
+        for(let j = a; i <= b; j++){
+            sum += j;
+        }
+    }
+    return sum;
```

Comment:
```suggestion
   for (let i = Math.min(a, b); i <= Math.max(a, b); i++) {
        sum += i;
    }
    
    return sum;
```
Math 함수를 사용하면 최소, 최대를 쉽게 구할 수 있으며, 반복문에 바로 적용하여 간단하게 작성할 수 있습니다.